src/metabase/lib/schema/template_tag.cljc

```diff
@@ -0,0 +1,132 @@
+(ns metabase.lib.schema.template-tag
```

Comment:
This stuff is all ported directly from the versions in `metabase.mbql.schema`; I didn't change anything other than translating from Malli